src/mlpack/methods/ann/rnn_impl.hpp

```diff
@@ -83,6 +83,15 @@ double RNN<OutputLayerType, InitializationRuleType, CustomLayers...>::Train(
     ResetParameters();
   }
 
+  if (optimizer.MaxIterations() < this->predictors.n_cols)
+  {
+    Log::Warn << "The number of iterations is less than the size of the "
+              << "dataset, the optimizer might not pass over the entire "
+              << "dataset. Please modify the number of iterations to be "
+              << "at least equal to the number of columns of your "
+              << "dataset." << std::endl;
```

Comment:
Hi @gaurav-singh1998, Could you also add the same message that you used for FNN here also. Thanks.

Reply:
Hi, @kartikdutt18 I was planning to do this but thought of first getting an assurance that what I did was correct. So can I move ahead and these test in `rnn` and `brnn` also?